frontend/app-development/features/processEditor/ProcessEditor.tsx

```diff
@@ -112,6 +116,7 @@ export const ProcessEditor = (): React.ReactElement => {
   return (
     <ProcessEditorImpl
       availableDataModelIds={availableDataModelIds}
+      currentPolicy={currentPolicy as Policy}
```

Comment:
Is there a reason we don't get this type from the query?

Reply:
The query type is from Policy Editor, and it has few more properties than needed in Process Editor, so I created an explicit Policy type for Process Editor, where I excluded the ` requiredAuthenticationLevelEndUser: RequiredAuthLevel;
  requiredAuthenticationLevelOrg: string;` properties.

Reply:
Fixed this casting by moving some types to app-shared. 😊